03_Further_Reading/README.md

```diff
@@ -0,0 +1,54 @@
+# Introduction
+
+Hope you enjoyed this Redux initial dive in, guess what? There's a very intersting ecosystem of third partie open source libraries that can add super powers to our redux 
```

Comment:
There's a very ~intersting~ **interesting** ecosystem of third ~partie~ **party**

Reply:
Good catch, corrected, thanks